extension/json/include/json_functions.hpp

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// json-extension.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+namespace duckdb {
+
+class JSONFunctions {
+public:
+	static vector<CreateScalarFunctionInfo> GetFunctions() {
+		auto functions = GetExtractFunctions();
```

Comment:
This should probably be moved into a `.cpp` file, I suspect this code being in a header is related to the linking failures.